Dockerfile

```diff
@@ -1,36 +1,32 @@
-FROM openjdk:8u121-jdk-alpine
+FROM openjdk:11-jdk as builder
```

Comment:
change this to
`FROM adoptopenjdk/openjdk11:jdk-11.0.6_10-alpine-slim as builder`

if you get some `--release flag` error, then remove the `-slim`:
`FROM adoptopenjdk/openjdk11:jdk-11.0.6_10-alpine as builder`